Bottom: 29d8762cbb07420dd0c2185e64828df55582ed84
Top:    170e1bd9f4bd26b57fbe9ec6d33684d47e1ad087
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-24 16:57:35 +0800

Refresh of vhost-device-iotlb-api

---

diff --git a/include/uapi/linux/vhost.h b/include/uapi/linux/vhost.h
index 19493ee..78f7d8f 100644
--- a/include/uapi/linux/vhost.h
+++ b/include/uapi/linux/vhost.h
@@ -68,6 +68,7 @@ struct vhost_msg {
   int type;
   union {
     struct vhost_iotlb_msg iotlb;
+    __u8 padding[64];
   };
 };
